packages/react-jsx-highstock/src/components/RangeSelector/RangeSelectorInner.js

```diff
@@ -1,103 +1,85 @@
-import React, { Component } from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { attempt } from 'lodash-es';
-import { Hidden, getModifiedProps, provideAxis } from 'react-jsx-highcharts';
+import { useHighcharts, useAxis, useChart, useModifiedProps } from 'react-jsx-highcharts';
 
-class RangeSelectorInner extends Component {
+const RangeSelectorInner = ({ enabled = true, axisId, children, ...restProps }) => {
```

Comment:
note: I think that RangeSelector and RangeSelectorInner could now be merged into single component

Reply:
I suspect you are correct - I believe the original reason for this was this need to [provide an axis](https://github.com/whawker/react-jsx-highcharts/blob/master/packages/react-jsx-highstock/src/components/RangeSelector/RangeSelectorInner.js#L100), but it looks as though with the hooks this shouldn't be necessary